updater/lib/tinglesoftware/azure/artifacts_credential_provider.rb

```diff
@@ -16,22 +18,26 @@ module Azure
     module ArtifactsCredentialProvider
       def self.install_if_nuget_feeds_are_configured
         credentials = JSON.parse(ENV.fetch("DEPENDABOT_EXTRA_CREDENTIALS", "[]"))
-        private_ado_nuget_feeds = credentials.select do |cred|
-          cred["type"] == "nuget_feed" && cred["url"].include?("dev.azure.com")
+        private_nuget_feeds = credentials.select do |cred|
+          cred["type"] == "nuget_feed"
         end
```

Comment:
We should exclude non-private feeds here such as public feeds for daily builds. Checking for presence or username/token should be enough.

Reply:
good point, forgot about public feeds